### DISCUSSION TITLE
How to make certain entities glow for certain players only?

### DISCUSSION THREAD

**Discussion by MrGazdag** [Minestom/Minestom#748] [Help]
I would like to make a player see an entity (specifically, an armor stand) as if it has `{Glowing:1b}`. How can I achieve this? I assume i need to work with `EntityMetadataPacket`s, but I hope there might be a better way than this.

**Comment by emortaldev** [comment]
I believe just managing packets is the only way to do it, the easiest atleast in my opinion. You can always make a wrapper for it
